Services/RequestServer/ConnectionFromClient.h

```diff
@@ -77,4 +77,6 @@ class ConnectionFromClient final
     NonnullRefPtr<Resolver> m_resolver;
 };
 
+constexpr inline uintptr_t websocket_private_tag = 0x1;
```

Comment:
wrong commit for this :)

Reply:
nuh, it's used in line 56 of the cpp <[`46383a7` (#3642)](https://github.com/LadybirdBrowser/ladybird/pull/3642/commits/46383a7965aab7ba5589314f8050c09c5c569a02#diff-56ad25be8d28d874fca6d3b309b7bddefc0336699218b6a8874cc81bb5a10450R56)>

I call `set_option(CURLOPT_PRIVATE, reinterpret_cast<uintptr_t>(this) | websocket_private_tag);` in this commit before adding it to the main connection from client code in the next commit.

Reply:
Ah! Missed that.
Nevermind then.